src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -71,7 +71,7 @@ static License GetTrialLicense()
             }
             else
             {
-                var message = $"Trial for Particular Service Platform is still active, trial expires on {trialLicense.ExpirationDate.Value.ToLocalTime().ToShortDateString()}.";
+                var message = $"Trial for the Particular Service Platform has been active since {trialStartDate.ToLocalTime().ToShortDateString()}.";
```

Comment:
I mention this as a trial conversion is very important from a business opportunity perspective.

This message is only written once at startup. It's not like that it is flooding the log file.

Don't forget that these can also be written to the windows event log. Messages with log level ERROR and CRITICAL (/FATAL) are likely to be investigated by operations. Such windows event log entries can contain a link and those will be clickable in there